zbarlight/__init__.py

```diff
@@ -1,5 +1,24 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
-from ._zbarlight import qr_code_scanner
+from ._zbarlight import zbar_code_scanner
+
+from PIL import Image
 
 __version__ = '0.1.1'
-__ALL__ = ['qr_code_scanner']
+__ALL__ = ['scan_codes', 'qr_code_scanner']
+
+
+def qr_code_scanner(image, width, height):
+    """Specific QR code scanner (deprecated)"""
+    result = zbar_code_scanner(b'qr.enable', image, width, height)
```

Comment:
If it is deprecated, perhaps we should emit a [deprecation warning](https://docs.python.org/2/library/exceptions.html#exceptions.DeprecationWarning)?